src/ui/styles/global.ts

```diff
@@ -0,0 +1,35 @@
+import { createGlobalStyle } from 'styled-components'
+import { theme } from './theme'
+
+export const GlobalStyle = createGlobalStyle`
+  * {
+    margin: 0;
+    padding: 0;
+    outline: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    background: ${theme.colors.light};
+    font-family: ${theme.font.family};
+    overflow-x: hidden;
+    color: ${theme.colors.dark};
+  }
+
```

Comment:
Podemos adicionar essa estilização para as fontes:

```suggestion
html {
    font-size: 62.5%;
}
```

Com ela vamos conseguir usar a medida rem de uma maneira mais simples, exemplo: 16px = 1.6rem, já que o rem é baseado no tamanho da font do html.